modules/main/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "luma.gl",
+  "version": "7.0.0-alpha.7",
+  "description": "WebGL2 Components for High Performance Rendering and Computation",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/uber/luma.gl"
+  },
+  "keywords": [
+    "webgl",
+    "visualization",
+    "animation",
+    "3d"
+  ],
+  "main": "dist/es5/index.js",
+  "module": "dist/esm/index.js",
+  "esnext": "dist/es6/index.js",
+  "files": [
+    "src",
+    "dist",
+    "addons.js",
+    "constants.js",
+    "debug.js",
+    "headless.js",
+    "webgl1.js",
+    "README.md"
+  ],
+  "scripts": {
+    "clean": "rm -fr dist dist-es6 && mkdir -p dist/es5 dist/esm dist/es6",
+    "build": "npm run clean && npm run build-es6 && npm run build-esm && npm run build-es5",
+    "build-es6": "BABEL_ENV=es6 babel src --config-file ../../babel.config.js --out-dir dist/es6 --source-maps --ignore 'node_modules/'",
+    "build-esm": "BABEL_ENV=esm babel src --config-file ../../babel.config.js --out-dir dist/esm --source-maps --ignore 'node_modules/'",
+    "build-es5": "BABEL_ENV=es5 babel src --config-file ../../babel.config.js --out-dir dist/es5 --source-maps --ignore 'node_modules/'",
+    "cover": "../../scripts/test.sh cover",
+    "prepublishOnly": "npm run build",
+    "publish-prod": "npm run build && npm run test-fast && npm publish",
```

Comment:
These should not be necessary:
`cover`
`prepublishOnly`
`publish-*`

Reply:
Removed in both places, thanks!